src/components/widgets/accounts-cash-projection/accounts-cash-projection.directive.coffee

```diff
@@ -103,6 +103,37 @@ module.controller('WidgetAccountsCashProjectionCtrl', ($scope, $q, $filter, $tim
       trxId
     ).then(-> $scope.trxList.updated = true)
 
+  $scope.trxList.includeSchedulableTransactions = (resourcesType, trx) ->
+    BoltResources.update(
+      w.metadata.bolt_path,
+      resourcesType,
+      trx.id,
+      {
+        recurring: trx.recurring,
+        recurring_pattern: trx.recurring_pattern,
+        recurring_end_date: if trx.recurring_end_date then moment(trx.recurring_end_date).format('YYYY-MM-DD') else null
+      }
+    ).then(->
+      $scope.trxList.updated = true
+      $scope.trxList.fetch()
+    )
+
+  $scope.trxList.deleteSchedulableTransactions = (resourcesType, trx) ->
```

Comment:
Do we really need a dedicated method for `deleteSchedulableTransactions`? I think it could be merged with `deleteTransaction`.

Reply:
No because I have to follow two different paths depends if the transactions is FORECAST or not. Actually I use deleteTransaction in one of the paths. :)